styles/conference/Purpose.module.less

```
@@ -0,0 +1,50 @@
+@media screen and (max-width:620px){
+    .abstract h2 {padding-top:34px;}
+    .abstract p{width: auto;}
+    .container{padding:0 10px;}
+}
+
+
+.abstract {
+    background: #F5F5F5;
+    //padding-bottom: 60px; 考虑用 pb-4 代替
+    // text-align: center; 考虑用 text-center
+
+
+    h2 {
+        margin: 0; // 可以考虑用 m-0 代替
+        font-size: 24px;
+        padding:70px 0 34px;
+    };
+
+    p { 
+        line-height: 1.6rem;
+        width: 80%;
+        margin: 0 auto;
+        text-align:left;
+    }
+
+    
+}
+
+.container{
+    max-width: 1200px;
+    margin: 0 auto;
+    font-size: 14px;
+    position:relative;
+    left:-50px;
+}
+
+.animated{
+    -webkit-animation:animated .5s ease-in-out forwards;
+}
+
+@-webkit-keyframes animated{
```

```suggestion
    animation: animated .5s ease-in-out forwards;
}

@keyframes animated {
```
现代 CSS 编译器能自动处理厂商兼容性前缀